### ISSUE TITLE
Add default configuration for MFA authenticator

### ISSUE THREAD

**Issue by ScopeyNZ** [silverstripe/silverstripe-mfa#9]
Currently users of this module should enable it by overriding the default authenticator provided by framework with the following config:

```
---
Name: mysitemfa
After: '#coresecurity'
---
SilverStripe\Core\Injector\Injector:
  SilverStripe\Security\Security:
    properties:
      Authenticators:
        default: %$SilverStripe\MFA\Authenticator\MemberAuthenticator
```

We should make this default configuration in this module so that there is no custom configuration required.

As there are no MFA methods as part of this module by default we will also have to ensure the whole MFA process is skipped given there are no MFA methods registered.

#### A/Cs:

- [x] Configuration is applied when this module is installed (#10)
- [x] Having no MFA methods installed & registered will skip the MFA part of login (#10)

**Comment by Cheddam** [comment]
I'm not sure this is the best approach, as in the long term this will likely break Framework tests in Kitchen Sink (once we have one or more MFA implementations applied to it).

Many other modules specify a small amount of config required to get them running (such as Elemental), so putting this config block in the README file should be sufficient. We can also manually apply this config in MFA's tests like so:

```
    public function setUp()
    {
        parent::setUp();

        Injector::inst()->load([
            'SilverStripe\Security\Security' => [
                'properties' => [
                    'authenticators' => [
                        'default' => '%$' . MemberAuthenticator::class,
                    ]
                ]
            ]
        ]);
    }
```

**Comment by ScopeyNZ** [comment]
Fair enough. I'll leave this parked for now then. I have still implemented half of this issue in a patch to #7 - the "Having no MFA methods installed & registered will skip the MFA part of login" A/C.

I've also added some more functional tests that assert that the new step is correctly inserted into the login flow.

See https://github.com/silverstripe/silverstripe-mfa/pull/10/commits/e8adc139f276d8ae4d8f70a8e400c0ecbd171076

**Comment by robbieaverill** [comment]
I like the idea of it being as easy as possible to get running. In that way, if it was enabled when installed but transparent until you have some authenticator modules added/configured as well, that seems like a good idea to me. You're right that our other authenticator modules (LDAP and RealMe) don't auto-configure themselves, but they also can't really do that because they both require external services to be configured etc. TOTP, U2F etc should be plug and play really.

**Comment by Cheddam** [comment]
Let's go ahead with this then - we can do a bit of work to isolate the framework tests when they break.

**Comment by ScopeyNZ** [comment]
Done in #10 🙂
